yb-voyager/src/srcdb/data/gather-assessment-metadata/oracle/object-type-mapping.sqlplus

```diff
@@ -16,15 +16,21 @@ SPOOL object-type-mapping.csv
 SELECT schema_name, object_name, object_type
 FROM (
     SELECT
-        owner AS schema_name,
-        -- TODO: handle the partitioned table support later
-        object_name || NVL2(subobject_name, '_' || subobject_name, '') AS object_name,
-        object_type
+        o.owner AS schema_name,
+        CASE
+            WHEN o.object_type = 'TABLE' THEN o.object_name
+            WHEN o.object_type = 'TABLE PARTITION' THEN o.object_name || '_' || o.subobject_name
+            WHEN o.object_type = 'TABLE SUBPARTITION' THEN o.object_name || '_' || sp.partition_name || '_' || o.subobject_name
+            ELSE o.object_name || NVL2(o.subobject_name, '_' || o.subobject_name, '')
```

Comment:
The else case seems to be for "INDEX PARTITION". What does that really mean? indexes also can be partitioned?

Reply:
In Oracle, the INDEX PARTITION object type refers to a partition of an index on a partitioned table. When a table is partitioned, its associated indexes can also be partitioned to improve performance and manageability. An INDEX PARTITION represents one partition of such an index.

Key Concepts
	1.	Partitioned Indexes: An index that is divided into smaller, more manageable pieces called partitions. Each partition can be managed and accessed independently.
	2.	Index Partitions: The individual pieces of a partitioned index. Each partition stores a subset of the index’s data.

Reply:
Sample Example of INDEX PARITITON:
```
CREATE INDEX order_status_idx on order_items_range_partitioned(order_status) LOCAL;
```

If we dont use the term `LOCAL` then it will be a GLOBAL index.

Global Index: It is not partitioned and span all the partitions of the table as a single structure. They do not get divided into smaller indexes for each partition.
Local Index: If you want to avoid the issues related to global indexes, you can create a local index. Local indexes are partitioned in alignment with the table partitions.